code/modules/client/preference/preferences.dm

```diff
@@ -184,8 +184,18 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 	var/slot_name = ""
 	var/saved = FALSE // Indicates whether the character comes from the database or not
 
-	// jukebox volume
-	var/volume = 100
+	/// Volume mixer, ordered by channel. Volume goes from 0 to 100.
+	var/list/volume_mixer = list(
+		"1024" = 100, // CHANNEL_LOBBYMUSIC
```

Comment:
Can't do `"[CHANNEL_ADMIN]"` at compile time so it has to be like this.

Additionally the keys are `/text` because `/number` results in out of bounds.